package.json

```diff
@@ -36,7 +36,7 @@
 		"@typescript-eslint/eslint-plugin": "^5.62.0",
 		"@typescript-eslint/parser": "^5.62.0",
 		"branch-name": "^1.0.0",
-		"browser-sync": "^3.0.3",
+		"browser-sync": "^3.0.4",
 		"eslint": "^8.57.1",
```

Comment:
@gnbm missing the virtual-select udpate on the dev dependency right?

Reply:
@BenOsodrac, I left a comment in the PR and in the 'What I Did' section. We basically aren’t able to do npm releases in Virtual Select since the token expired, and the repo owner is unresponsive. 
Because of this, there’s no version in npm to be updated 😕